frontend/packages/core/src/stepper.tsx

```diff
@@ -191,11 +192,22 @@ const Step: React.FC<StepProps> = ({ children }) => <>{children}</>;
 export interface StepperProps extends Pick<MuiStepperProps, "orientation"> {
   activeStep: number;
   children?: React.ReactElement<StepProps>[] | React.ReactElement<StepProps>;
+  nonLinear?: boolean;
+  completed?: { [key: number]: boolean };
```

Comment:
I'm not a huge fan of this property on the stepper. I think we should look at adjusting the `Step` component to take a property of `isCompleted` or something to that effect and let it manage whether it's been completed or not. Then we can have the step call into the wizard context which could keep the listing there.

Ultimately I want this to be more intuitive and allow the steps to define their own completion criteria that can then be propagated up versus having to maintain a list somewhere.

Reply:
Solved [here](https://github.com/lyft/clutch/pull/3071)